### PR TITLE
Merge dev to master

### PR DESCRIPTION
**Pull request by shadabwahidullah**
# In this milestone I have: 

- Structured the web app
- Designed components
- Added routing
- Connected to API
- Created state, store, reducers, and actions
- Added tests
- Deployed project on Netlify

## Used Technologies:

- CSS & HTML
- Bootstrap
- Javascript
- React
- Redux & thunk
- Jest

### CODE REVIEW

File: src/__test__/component.react.test.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  });
+  test('Navbar testing', () => {
+    const navbar = renderer.create(
+      <Provider store={store}>
+        <Navbar />
+      </Provider>,
+    ).toJSON;
+    expect(navbar).toMatchSnapshot();
+  });
+});
```
Comment (rukundoeric) [inline src/__test__/component.react.test.js:69]: - [x] <b>REQUIRED:</b> These test are not enough. Kindly add more test cases to comply with project requirements.
     - add unit tests for several components of the app and some pure functions in Redux code.
     - add integration tests using React Testing Library(`.toMatchSnapshot()` is not the only thing we need to check. you have to check the actual content of the component as well)

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hi @shadabwahidullah, 

_Great job so far 👏 💯 👍. However, there are few issues we need to address before we move on. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 

## Required Changes ♻️
  - [x] I could not find your video presentation of the project. The project requires to record a video presenting the features of the project:
       - The maximum length of the video is 5 minutes.
       - The video contains a clear and objective description about the project in professional English. The topic and purpose of the project are made clear.
       - The video contains a demo of the project, highlighting all the project features.
       - The video contains a highlight about a piece of the code the student finds interesting or is proud of making.
  - [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
  - [x] Check the comments under the review.

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/__test__/component.react.test.js (hunk 1 above)
Comment (shadabwahidullah) [reply src/__test__/component.react.test.js:69]: Ok, thanks for mentioning that

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hi there, 

Awesome job on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. Let's work on those.


 ## Required Changes ♻️
_Kindly check the comments under the review._


Happy coding, Cheers!🚀🚀

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it's valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    "ecmaVersion": 2018,
+    "sourceType": "module"
+  },
+  "extends": ["airbnb", "plugin:react/recommended"],
+  "plugins": ["react"],
+  "rules": {
+    "react/jsx-filename-extension": ["warn", { "extensions": [".js", ".jsx"] }],
+    "react/react-in-jsx-scope": "off",
+    "import/no-unresolved": "off",
+    "react/prop-types": "off",
```
Comment (mikenath223) [inline .eslintrc.json:21]: - [x] OPTIONAL - Please use the exact eslint linting rules found here. It's important you add proptypes to all components. Kindly remove the eslint rule disabling prop-types.

File: README.md [hunk 3]
```diff
@@ -1 +1,83 @@
-## Money Exchange
+# Money Exchange
+
+Photo gallery is a web app that is developed on react and provides the best free stock photos, royalty-free images & videos shared by creators.
+
+
+![video presentation](https://user-images.githubusercontent.com/51750930/143566841-63fb0571-90bd-4242-9635-5f930b6e1fe5.mp4)
```
Comment (mikenath223) [inline README.md]: - [x] Please record an additional video containing a highlight about a piece of code that you find interesting or are proud of making.

File: .eslintrc.json (hunk 2 above)
Comment (shadabwahidullah) [reply .eslintrc.json:21]: Thanks for pointing it out. I will make sure to do it.

File: README.md (hunk 3 above)
Comment (shadabwahidullah) [reply README.md]: Thank you, I will do it.

**Review by safafa — APPROVED** [review]
## STATUS: APPROVED 🟢 👏
Hello @shadabwahidullah  great job working on the required changes 👍 💯

Now you can merge this branch 💪 😄
HAPPY CODING!! AND GOOD LUCK WITH THE NEXT ONES 🤞 🍀

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**